ush/runWcoss.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
what about having one run script but it "sources" a machine specific file to do all of the module loads/paths/set APRUN, etc.?

Reply:
Yes, that could work.  The command file generated on wcoss2 would source the resource file on the first line then the plot commands would follow.  Note that there's nothing needed for hera -- the command file is just run directly with no extra fuss.

Reply:
I've replaced `runWcoss.sh `with `setup_wcoss2.sh`, which is now sourced in the first line of the `$cmdfile`.  `$cmdfile` is then directly run as the batch job on both hera and wcoss2.